### PR TITLE
Fix of $response->getBody()->getContents() being null when requested twice

### PR DESCRIPTION
**Pull request by murum**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 composer.lock
 phpunit.xml
 vendor
+.idea
```
Comment (vinkla) [inline .gitignore]: Please remove this. This should be added to your global ignore file.

**Comment by vinkla** [comment]
Thanks for your contribution. Please see my comments.

File: .gitignore (hunk 1 above)
Comment (murum) [inline .gitignore]: This is now fixed.

**Comment by vinkla** [comment]
Thanks!

**Pull request merged**